src/platform/Tizen/BLEManagerImpl.cpp

```diff
@@ -754,28 +750,24 @@ static bool __GattClientForeachCharCb(int total, int index, bt_gatt_h charHandle
 static bool __GattClientForeachServiceCb(int total, int index, bt_gatt_h svcHandle, void * data)
 {
     bt_gatt_type_e type;
-    char * uuid = nullptr;
-    auto conn   = static_cast<BLEConnection *>(data);
+    GAutoPtr<char> uuid;
+    auto conn = static_cast<BLEConnection *>(data);
     ChipLogProgress(DeviceLayer, "__GattClientForeachServiceCb");
 
-    VerifyOrExit(__GetAttInfo(svcHandle, &uuid, &type) == BT_ERROR_NONE,
-                 ChipLogError(DeviceLayer, "Failed to fetch GATT Attribute from SVC handle"));
+    VerifyOrReturnValue(__GetAttInfo(svcHandle, &MakeUniquePointerReceiver(uuid).Get(), &type) == BT_ERROR_NONE, true,
```

Comment:
I think that you should keep the logic with `VerifyOrExit`, because after the `exit:` label there is an explanation why we are returning `true` from that function (even in case of error).

Reply:
Yes you are right, thanks